### PR TITLE
Initial Commit for Flask Admin

### PR DESCRIPTION
**Pull request by lizzyaustad**
- transition from rails app to flask app
- create db for admin portal
- test send & receive with Africa's Talking api

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+3. Install requirements:
+
+    pip install -r 'requirements.txt'
+
+4. Run the application:
+
+    python3 app.py
+
+The first time you run this example, a sample sqlite database gets populated automatically. To suppress this behaviour,
+comment the following lines in app.py:
```
Comment (aashnisshah) [inline README.md]: add where in `app.py` to make this change

File: app.py [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+import africastalking
+
+import flask_admin as admin
+from flask_admin.contrib.sqla import ModelView
+
+app = Flask(__name__)
+app.config.from_pyfile('config.py')
+db = SQLAlchemy(app)
+
+username = "sandbox"
```
Comment (aashnisshah) [inline app.py]: At some point we'll want to extract these details into separate configs for each environment. @toriejw recently did it for the End Slavery Now Project

File: app.py [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+import flask_admin as admin
+from flask_admin.contrib.sqla import ModelView
+
+app = Flask(__name__)
+app.config.from_pyfile('config.py')
+db = SQLAlchemy(app)
+
+username = "sandbox"   
+api_key = os.environ.get('AT_API_KEY')   
+test_number = "+254723963007"
```
Comment (aashnisshah) [inline app.py]: is this a test number they provided?

File: app.py [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+def index():
+  if request.method == 'GET':
+    return '<a href="/admin/">Click me to get to Admin!</a>'
+  if request.method == 'POST':
+    msg = request.form.getlist('text')
+    cmd = msg[0].split()[0]
+    if cmd.lower() == 'loan':
+      add_transaction(request.form)
+      return 'success: added loan transaction'
+    # ImmutableMultiDict([
```
Comment (aashnisshah) [inline app.py]: What is this section for?

Comment (lizzyaustad) [reply app.py]: ah I just left that there to remind myself what the incoming messages look like to set up the parsing - I'll probably be removing them once I have the rest of that set up

File: app.py (hunk 3 above)
Comment (lizzyaustad) [reply app.py]: nope, their setup allows you to input any number for testing with the sandbox environment. I think I just grabbed that off wikipedia as an example number. I can make it more generic, I think it just has to have the 254 at the start

File: app.py (hunk 2 above)
Comment (lizzyaustad) [reply app.py]: sounds good - went ahead and added a ticket for this: https://github.com/elixirlabsinc/pangeanetwork/issues/6

File: README.md (hunk 1 above)
Comment (lizzyaustad) [reply README.md]: good call 👍 I'll probably wait a little bit to add that in while the app.py file still has a lot of incoming changes

File: app.py (hunk 3 above)
Comment (aashnisshah) [reply app.py]: Awesome :) while I'm in Kenya we can change that to my phone number for testing :)

**Comment by aashnisshah** [comment]
Awesome! Feel free to merge whenever you're ready - the changes look great!

**Pull request merged**